### ISSUE TITLE
Absolute path file name not generating file when using launch file

### ISSUE THREAD

**Issue by dmcconachie** [UM-ARM-Lab/arm_video_recorder#3]
Starting the service using the roslaunch file
```
 roslaunch arm_video_recorder arm_video_recorder.launch
```
and then calling the service
```
rosservice call /video_recorder  "filename: '/tmp/test_absolute.avi'
record: true
timeout_in_sec: 5.0" 
```
does not generate any file that I can find on my system.

Starting the service by just using `./video_recording.py` does generate a video at the expected `/tmp/test_absolute.avi`

**Comment by dmcconachie** [comment]
Related: if a `prefix_path` is set; it appears that that folder (i.e., `$(find arm_video_recorder)/Captures`) must exist for a file to be written, otherwise it silently fails to generate a file.

**Comment by bsaund** [comment]
Addressed in 68e64828820c9d03826fc435fad560cbe1d465e5
